README.rst

```diff
@@ -214,8 +214,8 @@ Choose from the list of available rules:
 
   Configuration options:
 
-  - ``syntax`` (``'long'``, ``'short'``): whether to use the ``long`` or ``short`` array
-    syntax; defaults to ``'long'``
+  - ``syntax`` (``'long'``, ``'short'``): whether to use the ``long`` or ``short`` array syntax;
```

Comment:
I couldn't find any reference to such limit. Anyway, changes in this file are only due to `` ` `` characters being replaced by coloring in the `fix --help` output and thus not taken into account in the line length calculation there.